dev/reference/mysql-compatibility.md

```diff
@@ -133,14 +134,14 @@ Create Table: CREATE TABLE `t1` (
 1 row in set (0.00 sec)
 ```
 
-从架构上讲，TiDB 确实支持类似 MySQL 的存储引擎抽象，在启动 TiDB（通常是 `tikv`）时 [`--store`](../sql/server-command-option.md#--store) 选项指定的引擎中创建用户表。
+从架构上讲，TiDB 确实支持类似 MySQL 的存储引擎抽象，在启动 TiDB（通常是 `tikv`）时 [`--store`](/sql/server-command-option.md#--store) 选项指定的引擎中创建用户表。
```

Comment:
Since you added an alias related to `server-command-option.md` (in op-guide/configuration.md → dev/reference/configuration/tidb-server/configuration.md), will it be necessary to update it here?

Reply:
No, it's redirected to the 配置参数 file as I commented above. And we could find the corresponding item `--store` there. So no need to update here.

Reply:
OK.